docker/Makefile

```diff
@@ -30,5 +30,5 @@ $(targets):
 
 push:
 	for name in $(targets); do \
-		podman push $(prefix)-$$name ovirtorg/$(prefix)-$$name; \
+		podman push $(prefix)-$$name ovirt/$(prefix)-$$name; \
```

Comment:
This push url is missing quay.io/ prefix, and should really be in its own commit.

Reply:
Right, I’ll separate it.